src/services/notifications/index.ts

```diff
@@ -446,58 +489,36 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     balance: CurrencyBaseAmount
     walletCurrency: WalletCurrency
     userId: UserId
-    price: DisplayCurrencyPerSat | ApplicationError
+    price: DisplayCurrencyPerSat | Error
```

Comment:
Why change from `ApplicationError`?

Reply:
ApplicationError is a type defined in app layer (service layer should not depend on app layer), but you are right, we have a problem here. what we really need is to receive a valid value for `displayCurrencyPerSat` so I changed the interface https://github.com/GaloyMoney/galoy/pull/1338/commits/4e304d4a71fd4eb5d775bf0de2d020d0efa013df#diff-efedf3aa6398865cf644d1c13a3f2a6248a28640237c7436f4c1db0b4d3ffdd1L56